components/text-input/examples/30-fieldset.js

```diff
@@ -12,31 +12,31 @@ function Example({ brand }) {
 			<h2>Default</h2>
 			<fieldset>
 				<legend>Legend</legend>
-				<TextInput />
+				<Text />
 				<br />
-				<TextInput tag="select">
+				<Select>
 					{options.map((v, i) => (
```

Comment:
Inside examples I would avoid loops that make it hard to read the code. We will likely use these examples on the website soon too.

Reply:
I'll fixed it up quickly